### ISSUE TITLE
exp always expires in 10 minutes regardless of expiresIn time

### ISSUE THREAD

**Issue by megmut** [auth0/node-jsonwebtoken#336]
When I generate a new JWT, it is always expiring in 10 minutes and 8 seconds. When consoling the iat and exp from the decoded token upon token generation, I can see that the difference between them is always 604800. 

Here is my token generation function:

`    
private static generateToken(username:string, admin:boolean, permissions:number):string {
    let data = {
        sub: username, 
        iss: 'mydomain',
        permissions: permissions,
        admin: admin
    }
    let token:string = null;
    try {
        token = jwt.sign(data, secret, {
            expiresIn: '7d'
        });
    } catch(err) {
        console.log(err);
    }
    return token;
}
`

It doesn't matter what I set the 'expiresIn' property to, it always expires in 10.8 minutes.

**Comment by ziluvatar** [comment]
@megmut that difference in seconds:

604800 seconds => 604800 / 60 sec per minute / 60 min per hour / 24 hour per day = 7 days

I haven't run the example though. How did you calculate the 10min and 8 seconds?

**Comment by jfromaniello** [comment]
You must be doing something wrong besides the code shown here. Check tests here:
https://github.com/auth0/node-jsonwebtoken/blob/master/test/expires_format.tests.js

**Comment by megmut** [comment]
Okay, sorry for the confusion, I calculated it in ms, not seconds. 
So if I console out the iat and exp when the token is decoded, I get this result
exp: 1491932757 iat:1491327957
Subtracting one from the other, I calculate 604800 (7 days), however if I start a timer and time verify after 15 minutes exactly, it was crashing. More and more debugging and it transpires I did something very stupid.

    router.post('/login', csrfProtection, auth.login, (req, res) => {
        res.cookie('authorization', res.locals.token, { maxAge: 900000, httpOnly: true });
        res.send({loggedIn: 'true', token:res.locals.token});
    });

turns out, the max age I was setting on the cookie to store the JWT is expiring in 15 minutes :(

I'm so sorry for the confusion! Feel free to close this, it's a non issue! :)
